en/admin/migration.inc

```diff
@@ -166,6 +166,14 @@ In general, the **cubrid loaddb** utility uses files (schema definition, object
 *   Trigger file(*database-name*\ **_trigger**): A file created by an unload operation. It contains information on the triggers defined in the database.
 *   User-defined object file(*user_defined_object_file*): A file in table format written by the user to enter mass data. (For details, see :ref:`howtowrite-loadfile` .)
 
```

Comment:
It may be useful to add a new paragraph to explain the order of processing input files when all are provided at once :).